### PR TITLE
12.0 c482b73d74502b6b1e4ed95a6bf96bb06d4e9c62

### PR DESCRIPTION
**Pull request by GabbasovDinar**

### CODE REVIEW

File: pos_multi_session/doc/changelog.rst [hunk 1]
```diff
@@ -1,5 +1,13 @@
+`4.2.2`
+-------
+
+**Fix:** `Longpoll Handling Error` related to property of undefined
+
```
Comment (Rusllan) [inline pos_multi_session/doc/changelog.rst:5]: Someone has to test it

**Review by Rusllan — COMMENTED** [review]
Some features needs to be tested

File: pos_orders_history/doc/changelog.rst [hunk 2]
```diff
@@ -3,6 +3,16 @@
...
+`1.1.0`
+-------
+
+- **NEW:** Update order list button
+
+`1.0.2`
+-------
+- **FIX:** displaying posted orders after open new POS session
+- **FIX:** error determining the current cashier
+
```
Comment (Rusllan) [inline pos_orders_history/doc/changelog.rst:15]: Someone has to test it

File: pos_orders_history_reprint/doc/changelog.rst [hunk 3]
```diff
@@ -3,6 +3,16 @@
...
+`1.0.2`
+-------
+
+- **IMP:** Improved a speed loading of receipts
+
+`1.0.1`
+-------
+
+- **FIX:** Compatibility with Point of Sale Receipt Reprinting
+
```
Comment (Rusllan) [inline pos_orders_history_reprint/doc/changelog.rst:15]: Someone has to test it

File: pos_qr_payments/doc/changelog.rst [hunk 4]
```diff
@@ -1,3 +1,8 @@
+`1.0.1`
+-------
+
+- **FIX** - Error related to pos reference in different translations
+
```
Comment (Rusllan) [inline pos_qr_payments/doc/changelog.rst:5]: Someone has to test it

**Comment by GabbasovDinar** [comment]
it will be done here https://github.com/it-projects-llc/pos-addons/pull/861

**Pull request closed without merging**